tasks/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 - name: Validate agent_type
   assert:
-    that: agent_type in ["monitoring"]
-    msg: "Received invalid agent type: '{{ agent_type }}'. The Cloud Ops Ansible role supports the following agents: 'monitoring', 'logging' and 'ops_agent'."
+    that: agent_type in ['monitoring', 'ops-agent']
```

Comment:
What about `logging`?

Reply:
That will be added in https://github.com/GoogleCloudPlatform/stackdriver-ansible-role/pull/41, I'll just have to merge the two accordingly.